blockchain/genesis/genesis.go

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) 2018 IoTeX
```

Comment:
Should perhaps update all copyright header to 2019 -- of course in another PR.

Reply:
But this should probably be 2019